doc/getting-started.md

```diff
@@ -37,67 +39,71 @@ To run Kubernetes on your local machine, we suggest
 #### `kubectl version --short`
 
 ### Which should display:
-```
+
+```bash
 Client Version: v1.10.3
 Server Version: v1.10.3
 ```
+
 Confirm that both `Client Version` and `Server Version` are v1.8.0 or greater.
 If not, or if `kubectl` displays an error message, your Kubernetes cluster may
 not exist or may not be set up correctly.
 
-___
+____
```

Comment:
Just curious, why make this change? I thought markdown only requires 3 underscores to create a horizontal rule.

Reply:
I got a markdown linter running to make sure I was wrapping at 80 chars and it appears to have auto-updated for me.